src/py/kaleido/scopes/plotly.py

```diff
@@ -59,6 +59,8 @@ def __init__(self, plotlyjs=None, mathjax=None, topojson=None, mapbox_access_tok
             'kaleido_scopes.js'
             )
         path = os.environ.get("BROWSER_PATH", which_browser())
+        if not path:
+            raise RuntimeError("Kaleido now requires that chrome/chromium is installed separately. Kaleido will try to detect it automatically, but the environmental error \"BROWSER_PATH\" can also be set")
```

Comment:
```suggestion
            raise RuntimeError("Kaleido now requires that chrome/chromium is installed separately. Kaleido will try to detect it automatically, but the environment variable \"BROWSER_PATH\" can also be set")
```

Reply:
wow okay this is correct why did I miss this? thank you